src/api/alert_words/removeAlertWord.js

```diff
@@ -0,0 +1,9 @@
+/* @flow strict-local */
+
+import type { ApiResponse, Auth } from '../transportTypes';
+import { apiDelete } from '../apiFetch';
+
+export default (auth: Auth, alertWord: Array<string>): Promise<ApiResponse> =>
```

Comment:
Yes, I thought `{add, remove}Alertword.js` is capable of adding/remove one-word on a single call. The overall result from these functions is the addition/deletion of a single word, whereas `getAlertWords` receive all the words. I thought this would explain very well even though it takes multiple words as an argument during addition/deletion.

Reply:
>The actual filenames should also be plural here ({add,remove}AlertWords.js), since the functions they export accept multiple items.

Yes, `{add, remove}Alertword.js` is capable of adding/remove one-word on a single call. The overall result of these functions is the addition/deletion of a single word. I thought this would explain very well even though it takes multiple words as an argument during addition/deletion.